src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -0,0 +1,333 @@
+        /// </param>
+        /// <param name="lpDependencies">
+        /// A pointer to a double null-terminated array of null-separated names of services or load ordering groups that the system must start before this service. Specify NULL or an empty string if the service has no dependencies. Dependency on a group means that this service can run if at least one member of the group is running after an attempt to start all members of the group.
+        /// You must prefix group names with SC_GROUP_IDENTIFIER so that they can be distinguished from a service name, because services and service groups share the same name space.
+        /// </param>
+        /// <param name="lpServiceStartName">
+        /// The name of the account under which the service should run.
+        /// If the service type is <see cref="ServiceType.SERVICE_WIN32_OWN_PROCESS"/>, use an account name in the form DomainName\UserName.
+        /// The service process will be logged on as this user.
+        /// If the account belongs to the built-in domain, you can specify .\UserName.
+        /// </param>
+        /// <param name="lpPassword">
+        /// The password to the account name specified by the lpServiceStartName parameter.
+        /// Specify an empty string if the account has no password or if the service runs in the LocalService, NetworkService, or LocalSystem account.
+        /// If the account name specified by the lpServiceStartName parameter is the name of a managed service account or virtual account name, the lpPassword parameter must be NULL.
+        /// Passwords are ignored for driver services.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the service.
+        /// If the function fails, the return value is NULL
+        /// </returns>
+        [DllImport(nameof(AdvApi32), SetLastError = true)]
+        public static extern IntPtr CreateService(IntPtr hSCManager, string lpServiceName, string lpDisplayName, ServiceAccess dwDesiredAccess, ServiceType dwServiceType, ServiceStartType dwStartType, ServiceErrorControl dwErrorControl, string lpBinaryPathName, string lpLoadOrderGroup, int lpdwTagId, string lpDependencies, string lpServiceStartName, string lpPassword);
+
+        /// <summary>
+        /// Marks the specified service for deletion from the service control manager database.
+        /// </summary>
+        /// <param name="hService">
+        /// A handle to the service. This handle is returned by the <see cref="OpenService"/> or <see cref="CreateService"/> function, and it must have the <see cref="ServiceAccess.Delete"/> access right
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is nonzero.
+        /// If the function fails, the return value is zero
+        /// </returns>
+        [DllImport(nameof(AdvApi32), SetLastError = true)]
+        public static extern IntPtr DeleteService(IntPtr hService);
```

Comment:
DeleteService return `bool` according to https://msdn.microsoft.com/en-us/library/windows/desktop/ms682562(v=vs.85).aspx